torchvision/prototype/features/_image.py

```diff
@@ -18,6 +18,15 @@ class Image(Feature):
     color_spaces = ColorSpace
     color_space: ColorSpace
 
+    @classmethod
+    def _to_tensor(cls, data, *, dtype, device):
+        tensor = torch.as_tensor(data, dtype=dtype, device=device)
+        if tensor.ndim == 2:
+            tensor = tensor.unsqueeze(0)
+        elif tensor.ndim != 3:
+            raise ValueError()
```

Comment:
So batch of images should be handled separately?

Reply:
Isn't that what we agreed to? Right now there is no plan to add a special feature for `BatchImage` or the like.

Reply:
I think we might be confusing things. Probably better to discuss on VC

Reply:
As discussed offline, we will not a `BatchedImages` feature for now.